<launch>

    <param name="use_sim_time" value="true" />
    <node pkg="rviz" type="rviz" name="display_result" output="screen" args="-d $(find localization)/config/localiztion.rviz" />

    <param name="map_path" type="string" value="$(find localization)/data/pcd_tiles_filtered/" />
    <param name="save_path" type="string" value="$(find localization)/result/result2.csv" />


    
    <node pkg="localization" type="map_pub" name="map_punlisher" output="screen"/>
    
    <node pkg="localization" type="radar" name="radar_subscriber" output="screen">
        <param name="intensity_thres" type="double" value="90"/>    
        <!-- <param name="intensity_thres" type="double" value="85"/> -->
        
    </node>
    <node pkg="localization" type="localization" name="localizer" output="screen">
        <param name="low_pass_filter_gain" type="double" value="0.88"/>
        <param name="low_pass_filter_angle_gain" type="double" value="1.0"/>
    </node>
    
    <arg name="bags" default="
        $(find localization)/data/sdc_track2.bag
    "/>

    <node pkg="rosbag" type="play" name="bag_player" required="true" output="screen"
        args="
        --clock
        --pause
        -r 0.12
        --bags $(arg bags)
    "/>
</launch>